### PR TITLE
Update YAML files

### LINKED ISSUE #541: Synchronizing YAML files
The Cake.AddinDiscoverer tool has discovered discrepencies between the YAML files currently on Cake's web site and the packages discovered on Nuget.org:

YAML files to be deleted:

YAML files to be created:
- Cake.Android.AvdManager
- Cake.APT.Module
- Cake.Aws.ElasticBeanstalk.Dev
- Cake.AWS.Lambda
- Cake.AzCopy
- Cake.Bower
- Cake.Bridge
- Cake.BuildSystems.Module
- Cake.CakeBoss
- Cake.Chocolatey.Module
- Cake.CK.Pack
- Cake.ClickTwice
- Cake.ConventionalChangelog
- Cake.Deploy.Azure.Authentication
- Cake.Deploy.Bot.LUIS
- Cake.Deploy.Variables
- Cake.DNF.Module
- Cake.DocCreator
- Cake.DocumentDb
- Cake.Eazfuscator.Net
- Cake.Flyway
- Cake.Frosting.Template
- Cake.FtpDeploy
- Cake.Git.NonPortable
- Cake.Gradle
- Cake.Graph
- Cake.Handlebars
- Cake.HandlebarsDotNet
- Cake.Helm
- Cake.HgVersion
- Cake.Jira
- Cake.Karma
- Cake.LongPath.Module
- Cake.Mage
- Cake.Markdown-Pdf
- Cake.MonoApiTools
- Cake.Netlify
- Cake.NewRelic
- Cake.Npx
- Cake.NSpec
- Cake.NSwag.Console
- Cake.NuGet
- Cake.OssIndex
- Cake.Paket.Module
- Cake.Parallel.Module
- Cake.PinNuGetDependency
- Cake.Recipe
- Cake.ScheduledTasks
- Cake.Scripting.Abstractions
- Cake.Scripting.Transport
- Cake.ServiceOrchestration
- Cake.SimpleHTTPServer
- Cake.Storm.Android
- Cake.Storm.Environment
- Cake.Storm.Fluent
- Cake.Storm.Fluent.Android
- Cake.Storm.Fluent.DotNetCore
- Cake.Storm.Fluent.iOS
- Cake.Storm.Fluent.NuGet
- Cake.Storm.Fluent.Transformations
- Cake.Storm.iOS
- Cake.Storm.JsonBuildConfiguration
- Cake.Storm.Nuget
- Cake.Testing
- Cake.Tin
- Cake.TravisCI
- Cake.UncUtils
- Cake.UrlLoadDirective.Module
- Cake.UServer
- Cake.ViewCompiler
- Cake.VSforMac
- Cake.WindowsAppStore
- Cake.Wyam.Recipe
- Cake.Xamarin.Build
- Cake.Xamarin.Build.CakeBuilder

YAML files to be updated:
- Cake.ActiveDirectory
- Cake.AliaSql
- Cake.Apigee
- Cake.AppleSimulator
- Cake.AWS.CloudFront
- Cake.AWS.CodeDeploy
- Cake.AWS.EC2
- Cake.AWS.ElasticLoadBalancing
- Cake.AWS.Route53
- Cake.AWS.S3
- Cake.Bumpy
- Cake.Chutzpah
- Cake.CMake
- Cake.CodeAnalysisReporting
- Cake.Compression
- Cake.Coveralls
- Cake.DocFx
- Cake.Docker
- Cake.DotNetCoreEf
- Cake.EntityFramework
- Cake.ExtendedNuGet
- Cake.FluentMigrator
- Cake.Gem
- Cake.Genymotion
- Cake.Git
- Cake.Gitter
- Cake.Grunt
- Cake.Gulp
- Cake.HipChat
- Cake.HockeyApp
- Cake.Hosts
- Cake.Http
- Cake.IIS
- Cake.ImageOptimizer
- Cake.Issues
- Cake.Issues.DocFx
- Cake.Issues.EsLint
- Cake.Issues.InspectCode
- Cake.Issues.Markdownlint
- Cake.Issues.MsBuild
- Cake.Issues.PullRequests
- Cake.Issues.PullRequests.Tfs
- Cake.Issues.Reporting
- Cake.Issues.Reporting.Generic
- Cake.Issues.Testing
- Cake.Kudu
- Cake.Kudu.Client
- Cake.MicrosoftTeams
- Cake.MobileCenter
- Cake.MSBuildTask
- Cake.MsDeploy
- Cake.Npm
- Cake.OctoDeploy
- Cake.Openshift
- Cake.Paket
- Cake.Plist
- Cake.Powershell
- Cake.ProjHelpers
- Cake.Putty
- Cake.Raygun
- Cake.ReSharperReports
- Cake.Services
- Cake.Slack
- Cake.SqlServer
- Cake.SqlTools
- Cake.Terraform
- Cake.Tfx
- Cake.Topshelf
- Cake.Twitter
- Cake.VersionReader
- Cake.VsCode
- Cake.VsixSignTool
- Cake.VsMetrics
- Cake.WebDeploy
- Cake.Webpack

### PR DESCRIPTION
**Pull request by Jericho**
Resolves #541

### CODE REVIEW

**Comment by Jericho** [comment]
AppVeyor build failed:
```
Cake.exe : Package 'Cake.Frosting.Template 0.1.0-alpha0070' has a package type 'Template' that is not supported by project 'C:/projects/website/release/addins'.
At line:1 char:1
+ & "C:\projects\website\tools\Cake\Cake.exe" "build.cake" -target="App ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Package 'Cake.F...elease/addins'.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
```
I'm not sure what this exception means. Does it mean that we shouldn't generate YAML files for packages containing recipes?

File: addins/Cake.APT.Module.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Name: Cake.APT.Module
```
Comment (gep13) [inline addins/Cake.APT.Module.yml]: @Jericho I thought we weren’t including modules?

Comment (Jericho) [reply addins/Cake.APT.Module.yml]: I know, I forgot.

**Comment by gep13** [comment]
@Jericho it is a fairly safe bet that you can add that project to the blacklist that you have.

**Comment by Jericho** [comment]
just this particular one? or any package containing "recipes" ?

**Comment by Jericho** [comment]
and I want to make sure we are on same page: the blacklist I currently maintain excludes certain packages from the whole audit process, not just yml generation.

So when you say that I should add this package to my blacklist, do you really mean that this package should be excluded from the audit?

**Comment by gep13** [comment]
Yes, I think so. Can you link me to where you are maintaining that blacklist?

File: addins/Cake.APT.Module.yml (hunk 1 above)
Comment (gep13) [reply addins/Cake.APT.Module.yml]: 😄

**Comment by Jericho** [comment]
It's hard coded at the moment:
```csharp
// This is a hardcoded list of addins that we specifically want to exclude from our reports
private readonly string[] _blackListedAddins = new string[]
{
    "Cake.Bakery",
    "Cake.Common",
    "Cake.Core",
    "Cake.CoreCLR",
    "Cake.Email.Common"
};
```

**Comment by Jericho** [comment]
I looked at the content of the `.nupkg` and I tihnk I understand what's different about this package: it contains c# code template. It' does not contain recipes.

Therefore I agree with you that this package should be completely excluded from the audit process.

**Comment by Jericho** [comment]
I'm closing this PR and its related issue so I can run the audit process again and generate a new PR.

**Comment by gep13** [comment]
Why does Cake.Grunt appear in the list of recipes?

**Comment by Jericho** [comment]
In order for the audit process to identify a package as containing an addin, we must find a DLL in the root folder of the package, or in the `bin` folder (or a sub-folder), or in the `lib` folder (or a sub-folder).

In the case of Cake.Grunt, the DLL is located in a folder called "netstandard2.0" directly under the root. I have never seen this folder structure before.

**Pull request closed without merging**